downstream/modules/platform/con_storage-options-for-operator-installation-on-ocp.adoc

```diff
@@ -0,0 +1,14 @@
+[id="con_storage-options-for-operator-installation-on-ocp_{context}"]
+= Storage options for {OperatorPlatform} installation on {OCP}
```

Comment:
s/options/requirements 

I don't think RWX storage is optional for the hub.

Reply:
I admit I have a technological gap here with this product. Are you referring to the title for this? The SME, Chris Hambridge, reviewed this a couple of weeks ago and did not mention the title being erroneous. That said, it doesn't mean there might be a better way of describing the content. I'll have to circle back to check on this, but he didn't mention any issues with the title. Going with "stet" with this one for now.